filament/backend/src/vulkan/VulkanFboCache.cpp

```diff
@@ -288,16 +308,19 @@ VkRenderPass VulkanFboCache::getRenderPass(RenderPassKey config) noexcept {
         const bool clear = any(config.clear & TargetBufferFlags::DEPTH);
         const bool discardStart = any(config.discardStart & TargetBufferFlags::DEPTH);
         const bool discardEnd = any(config.discardEnd & TargetBufferFlags::DEPTH);
+        const VkAttachmentLoadOp loadOp = clear ? kClear : (discardStart ? kDontCare : kKeep);
         depthAttachmentRef.layout = imgutil::getVkLayout(VulkanLayout::DEPTH_ATTACHMENT);
         depthAttachmentRef.attachment = attachmentIndex;
+        VkImageLayout initialLayout = imgutil::getVkLayout(config.initialDepthLayout);
```

Comment:
why make these changes here? no logic seems to have changed.

Reply:
Removed. Sorry, was part of an earlier  revision.